### PR TITLE
Version 0.6.0

### PR DESCRIPTION
**Pull request by dkhalanskyjb**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+- Introduce the widely requested API for locale-invariant parsing and formatting ([#343](https://github.com/Kotlin/kotlinx-datetime/pull/343))
+- Completely overhaul the KDoc-based documentation ([#347](https://github.com/Kotlin/kotlinx-datetime/issues/347))
+- Breaking change: forbid parsing `Instant` values without the second-of-minute component on the JVM and JS, fixing inconsistency with Native ([#369](https://github.com/Kotlin/kotlinx-datetime/issues/369))
+- Breaking change: use the fully qualified name of classes in the JSON serializers ([#308](https://github.com/Kotlin/kotlinx-datetime/pull/308))
+- Fix Proguard emitting warning about missing `kotlinx-serialization` classes when serialization is not used ([#336](https://github.com/Kotlin/kotlinx-datetime/pull/336))
+- Reimplement the timezone database handling for Native targets from scratch ([#286](https://github.com/Kotlin/kotlinx-datetime/pull/286), [#327](https://github.com/Kotlin/kotlinx-datetime/pull/327))
+- Support Android NDK targets ([#344](https://github.com/Kotlin/kotlinx-datetime/pull/344))
+- Small tweaks and fixes.
+
+### Changelog relative to version 0.6.0-RC.2
```
Comment (ilya-g) [inline CHANGELOG.md:14]: I believe we can squash all pre-release versions between 0.5.0 and 0.6.0

**Review by ilya-g — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dkhalanskyjb) [reply CHANGELOG.md:14]: What do you mean? The changelog relative to 0.5.0 is above this line.

Comment (ilya-g) [reply CHANGELOG.md:14]: I mean, remove entries for 0.6.0-RC1, RC2 and so on

Comment (dkhalanskyjb) [reply CHANGELOG.md:14]: I think `Changelog relative to version 0.6.0-RC.2` could still be useful for our early adopters. We can remove the `RC` versions from the changelog after we publish one more version after `0.6.0`, when they are no longer relevant.

**Pull request merged**